src/main/java/org/abstractj/kalium/NaCl.java

```diff
@@ -24,105 +24,130 @@
+				@In byte[] publicKey, @In byte[] privateKey);
 
-        public int crypto_box_curve25519xsalsa20poly1305_open(@Out byte[] message, @In byte[] ct, @u_int64_t long length,
-                                                              @In byte[] nonce, @In byte[] publicKey, @In byte[] privateKey);
+		public int crypto_box_curve25519xsalsa20poly1305_open(
+				@Out byte[] message, @In byte[] ct, @u_int64_t long length,
+				@In byte[] nonce, @In byte[] publicKey, @In byte[] privateKey);
 
-        public static final int SCALAR_BYTES = 32;
+		public static final int SCALAR_BYTES = 32;
 
-        public int crypto_scalarmult_curve25519(@Out byte[] result, @In byte[] intValue, @In byte[] point);
+		public int crypto_scalarmult_curve25519(@Out byte[] result,
+				@In byte[] intValue, @In byte[] point);
 
-        public static final int XSALSA20_POLY1305_SECRETBOX_KEYBYTES = 32;
-        public static final int XSALSA20_POLY1305_SECRETBOX_NONCEBYTES = 24;
+		public static final int XSALSA20_POLY1305_SECRETBOX_KEYBYTES = 32;
+		public static final int XSALSA20_POLY1305_SECRETBOX_NONCEBYTES = 24;
 
-        int crypto_secretbox_xsalsa20poly1305(@Out byte[] ct, @In byte[] msg, @u_int64_t long length, @In byte[] nonce, @In byte[] key);
+		int crypto_secretbox_xsalsa20poly1305(@Out byte[] ct, @In byte[] msg,
+				@u_int64_t long length, @In byte[] nonce, @In byte[] key);
 
-        int crypto_secretbox_xsalsa20poly1305_open(@Out byte[] message, @In byte[] ct, @u_int64_t long length, @In byte[] nonce, @In byte[] key);
+		int crypto_secretbox_xsalsa20poly1305_open(@Out byte[] message,
+				@In byte[] ct, @u_int64_t long length, @In byte[] nonce,
+				@In byte[] key);
 
-        public static final int SIGNATURE_BYTES = 64;
+		public static final int SIGNATURE_BYTES = 64;
 
-        int crypto_sign_ed25519_seed_keypair(@Out byte[] publicKey, @Out byte[] secretKey, @In byte[] seed);
+		int crypto_sign_ed25519_seed_keypair(@Out byte[] publicKey,
+				@Out byte[] secretKey, @In byte[] seed);
 
-        int crypto_sign_ed25519(@Out byte[] buffer, @Out LongLongByReference bufferLen, @In byte[] message, @u_int64_t long length, @In byte[] secretKey);
+		int crypto_sign_ed25519(@Out byte[] buffer,
+				@Out LongLongByReference bufferLen, @In byte[] message,
+				@u_int64_t long length, @In byte[] secretKey);
 
-        int crypto_sign_ed25519_open(@Out byte[] buffer, @Out LongLongByReference bufferLen, @In byte[] sigAndMsg, @u_int64_t long length, @In byte[] key);
-    }
+		int crypto_sign_ed25519_open(@Out byte[] buffer,
+				@Out LongLongByReference bufferLen, @In byte[] sigAndMsg,
+				@u_int64_t long length, @In byte[] key);
 
-    /**
-     * This function isn't thread safe. Be sure to call it once, and before performing other operations.
-     *
-     * Check libsodium's <i>sodium_init()</i> documentation for more info.
-     */
-    public static int init() {
-        return sodium().sodium_init();
-    }
```

Comment:
Correct me if I'm wrong, but the only changes that really matters to this PR is the AES 128 support, is the code below.
Other than that, the changes above are not relevant, because they are pretty much trailing spaces.

Also please make sure to follow the formatting from the project.